### PR TITLE
[ja] translate 3-0-migration-guide.rst

### PR DESCRIPTION
**Pull request by okinaka**

### CODE REVIEW

File: ja/appendices/3-0-migration-guide.rst [hunk 1]
```diff
@@ -1,13 +1,1321 @@
...
+============
+
+シェルとタスクは ``Console/Command`` と ``Console/Command/Task`` から ``Shell`` と
+``Shell/Task`` に移動されました。
+
+ApiShell を削除
+----------------
+
+それ自身のファイルソースに何らかの利益やオンラインドキュメント/
+`API <http://api.cakephp.org/>`_ を提供しなかったとして ApiShell は削除されました。
```
Comment (chinpei215) [inline ja/appendices/3-0-migration-guide.rst]: > as it didn't provide any benefit over the file source itself and the online documentation

I think `any benefit over ...` should be translated as `...以上の何の利益`.

ファイルソース自体とオンラインドキュメンテーション以上の何の利益も提供しなかったので

File: ja/appendices/3-0-migration-guide.rst [hunk 2]
```diff
@@ -1,13 +1,1321 @@
...
+
+ルーティング
+============
+
+名前付きパラメータ
+------------------
+
+名前付きパラメータは 3.0 で削除されました。名前付きパラメータは、クエリ文字列パラメータの
+「きれいな」バージョンとして 1.2.0 で追加されました。
+視覚的な利点は議論の余地がありますが、名前付きパラメータが引き起こした問題ではありません。
```
Comment (chinpei215) [inline ja/appendices/3-0-migration-guide.rst]: > While the visual benefit is arguable, the problems named parameters created are not.

I think `are not` would mean `are not (arguable)`.

視覚的な利点は議論の余地がありますが、名前付きパラメータが引き起こした問題には議論の余地はありません。

**Comment by chinpei215** [comment]
There are few incorrect translations. But great work!

**Comment by okinaka** [comment]
@chinpei215 Thank you for your review! 
I fixed these issue.

**Comment by chinpei215** [comment]
@okinaka Thanks!

**Pull request merged**